src/clients/api/queries/getLegacyPool/useGetLegacyPool.ts

```diff
@@ -77,7 +78,7 @@ const useGetLegacyPool = (input?: TrimmedInput, options?: Options) => {
     (options?.enabled === undefined || options?.enabled);
 
   return useQuery(
-    [FunctionKey.GET_LEGACY_POOL, { ...input, chainId }],
+    options?.queryKey ? options.queryKey : [FunctionKey.GET_LEGACY_POOL, { ...input, chainId }],
```

Comment:
It seems odd to me that the query key would need to change. Why not pass the accountAddress and use that to create they query key?

Reply:
The thing here is that this hook is also used by the ClaimRewards button, which refreshes the key with accountAddress every few seconds. This means that the calculator also gets refreshed values, re-rendering itself. So the intention here was to have a designated key for the calculator alone, so it wouldn't use the same as ClaimRewards.